packages/lib/src/vis/tiles/utils.ts

```diff
@@ -50,32 +47,51 @@ export function sortTilesByDistanceTo(
   });
 }
 
-export function getScaledVisibleDomains(
+const NDC_BOX = new Box3(new Vector3(-1, -1, -1), new Vector3(1, 1, 1));
+
+function getObject3DVisibleBox(
```

Comment:
This returns the bounding box in the object's coordinates of the threejs displayed box